lib/Dialect/FIRRTL/InnerSymbolTable.cpp

```diff
@@ -118,9 +118,9 @@ StringAttr InnerSymbolTable::getInnerSymbol(InnerSymTarget target) {
     // TODO: update this when ports support per-field symbols
     auto sym = mod.getPortSymbolAttr(target.getPort());
     // Workaround quirk with empty string for no symbol on ports.
-    if (sym && sym.getValue().empty())
+    if (!sym)
       return {};
-    return sym;
+    return sym.getSymName();
```

Comment:
This needs to accomodate the field as well.  Previously the `isPort()` if/else set `base` (the `InnerSymAttr`) on both branches, and fell through to grabbing whatever field is targeted (as it presently done for non-ports), but simplified per style rules into current form.
If this description is unclear I can update this part if you'd like!

Reply:
Thanks for explaining it. used the fieldID.